participants/scipy_150/doc/README.md

```diff
@@ -1,5 +1,4 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+There are some functions that do stuf and thats:
```

Comment:
I still think this documentation is a little bit confusing. What do you think about saying what the function does, rather than just "stuff"?